WalletWasabi/CoinJoin/Client/Clients/AliceClientBase.cs

```diff
@@ -173,11 +173,11 @@ public async Task PostUnConfirmationAsync()
 				{
 					return;
 				}
-				catch (TorConnectionException) // If some internet connection issue then it'll likely time out and take it as unconfirmed.
+				catch (HttpRequestException ex) when (ex.InnerException is TorConnectionException) // If some Internet connection issue then it'll likely time out and take it as unconfirmed.
```

Comment:
These `catch` blocks look ugly. I think we could at least unify them with:

```c#
				catch (HttpRequestException ex) when (ex.InnerException is TorConnectionException or TorConnectCommandFailedException)
```

What it is the same:


```c#
				catch (HttpRequestException ex) when (ex.InnerException is TorException)
```

Reply:
Good point